### PR TITLE
chore: scripts to audit on-chain contract setup

### PR DESCRIPTION
**Pull request by kupermind**
- Adding scripts to audit on-chain contract setup.

### CODE REVIEW

File: abis/0.8.19/GovernorOLAS.json [hunk 1]
```diff
@@ -1,1425 +0,0 @@
-{
```
Comment (kupermind) [inline abis/0.8.19/GovernorOLAS.json:1]: This ABI is not used anywhere in contracts or setup, it was the intermediary one

File: hardhat.config.js [hunk 2]
```diff
@@ -106,7 +106,7 @@ module.exports = {
                 settings: {
                     optimizer: {
                         enabled: true,
-                        runs: 750,
+                        runs: 1000000,
```
Comment (kupermind) [inline hardhat.config.js:109]: Restoring the justice!

File: scripts/audit_chains/audit_contracts_setup.js [hunk 3]
```diff
@@ -0,0 +1,385 @@
...
+    const timelock = await governor.timelock();
+    customExpect(timelock, globalsInstance["timelockAddress"], log + ", function: timelock()");
+
+    // Check version, hardcoded
+    const version = await governor.version();
+    customExpect(version, "1", log + ", function: version()");
+
+    // Check quorumNumerator, hardcoded
+    const quorumNumerator = await governor["quorumNumerator()"]();
+    customExpect(quorumNumerator.toString(), globalsInstance["quorum"], log + ", function: quorumNumerator()");
```
Comment (mariapiamo) [inline scripts/audit_chains/audit_contracts_setup.js:211]: I would have expected that this and the the following two would have shown an issue. Indeed governorOLAS on goerli have different `quorum`, `initialVotingDelay`,  `initialVotingPeriod`. I guess that hardcoded values should be used here

Comment (kupermind) [reply scripts/audit_chains/audit_contracts_setup.js:211]: We compare values with our globals files - they are different for mainnet and goerli. If we want this to be exactly as on the mainnet, we need to use its values then.

Comment (kupermind) [reply scripts/audit_chains/audit_contracts_setup.js]: Updated with the mainnet values.

**Review by mariapiamo — APPROVED** [review]

**Pull request merged**